pom.xml

```diff
@@ -225,7 +225,7 @@
             <dependency>
                 <groupId>com.google.guava</groupId>
                 <artifactId>guava</artifactId>
-                <version>21.0</version>
```

Comment:
Guava versioning and backwards compatibility is a mess. I'm somewhat concerned we'll get some random downstream breakage because we changed this version.